sdk/maps/azure-maps-search/tests/test_maps_search_client.py

```diff
@@ -20,235 +24,83 @@ def setup_method(self, method):
-    @recorded_by_proxy
-    def test_search_nearby_point_of_interest(self):
-        result = self.client.search_nearby_point_of_interest(coordinates=(25.0338053, 121.5640089))
-        assert len(result.results) > 0
-        for item in result.results:
-            assert item.type == "POI"
+        assert len(result2.features) == 1
+        coordinates2 = result2.features[0].geometry.coordinates
+        longitude2 = coordinates2[0]
+        latitude2 = coordinates2[1]
 
-    @MapsSearchPreparer()
-    @recorded_by_proxy
-    def test_search_point_of_interest_category(self):
-        result = self.client.search_point_of_interest_category(
-            "RESTAURANT", coordinates=(25.0338053, 121.5640089)
-        )
-        assert len(result.results) > 0
-        for item in result.results:
-            assert item.type == "POI"
-            assert "RESTAURANT" in [category.code for category in item.point_of_interest.classifications]
+        assert longitude2 == -122.138669
+        assert latitude2 == 47.630359
 
     @MapsSearchPreparer()
     @recorded_by_proxy
-    def test_search_structured_address(self):
-        addr = StructuredAddress(street_number="221",
-                                 street_name="Sec. 2, Zhishan Rd.",
-                                 municipality_subdivision="Shilin Dist.",
-                                 municipality="Taipei City",
-                                 country_code="TW")
-        result = self.client.search_structured_address(addr)
-        assert len(result.results) > 0
-        for item in result.results:
-            assert item.type != "POI"
-        top_answer = result.results[0]
-        assert top_answer.type == "Point Address"
-        assert top_answer.address.street_name == "Zhishan Road Section 2"
-        assert top_answer.address.municipality == "Taipei City"
-        assert top_answer.address.postal_code == "111"
-        assert top_answer.address.country_code_iso3 == "TWN"
+    def test_reverse_geocode(self):
+        result = self.client.get_reverse_geocoding(coordinates=[-122.138679, 47.630356])
+        assert len(result.features) == 1
+        address = result.features[0].properties.address
+        assert address.formatted_address == "2265 152nd Ave NE, Redmond, WA 98052, United States"
 
     @MapsSearchPreparer()
     @recorded_by_proxy
-    def test_get_geometries(self):
-        result = self.client.get_geometries(geometry_ids=["8bceafe8-3d98-4445-b29b-fd81d3e9adf5"])
-        assert len(result) > 0
-        assert result[0].provider_id == "8bceafe8-3d98-4445-b29b-fd81d3e9adf5"
-        top_answer = result[0]
-        assert top_answer.geometry_data.type == "FeatureCollection"
-        assert len(top_answer.geometry_data.features) > 0
+    def test_reverse_geocode_batch(self):
+        coordinates1 = ReverseGeocodingBatchRequestItem(coordinates=[-122.138679, 47.630356])
+        coordinates2 = ReverseGeocodingBatchRequestItem(coordinates=[-122.138679, 47.630356])
```

Comment:
Can we use different coordinates for batch APIs in the tests? Same for other batch testing & samples.